scripts/build_test.py

```diff
@@ -91,8 +93,10 @@ def test_ensure_files_exist(self):
         """
         non_existent_filepaths = [INVALID_INPUT_FILEPATH]
         # Exception will be raised at first file determined to be non-existent.
+        error_message = ('File %s does not exist.') % non_existent_filepaths[0]
+        error_message = re.escape(error_message)
```

Comment:
Perhaps you can explain in a comment why this is necessary...

Reply:
Updated